src/ape/contracts/base.py

```diff
@@ -757,6 +757,27 @@ def __init__(
         self.txn_hash = txn_hash
         self._cached_receipt: Optional[ReceiptAPI] = None
 
+    def __call__(self, *args, **kwargs) -> ReceiptAPI:
```

Comment:
Nice, I like this

And can just bypass this check with `contract.as_transaction()`

Reply:
- [x] Show in tests how to bypass checks

^ Just wanted to make sure that actually worked. It does!